app/models/variant.rb

```diff
@@ -23,6 +23,12 @@ class Variant < ActiveRecord::Base
 
   enum reference_build: [:GRCh38, :GRCh37, :NCBI36]
 
+  after_initialize :init
+
+  def init
+    self.civic_actionability_score = 0
```

Comment:
Would someone ever want to set an explicit value during creation that we need to avoid stomping?  (Don't see any in the codebase currently.)  If so `||=` would go well here.